static/include/components/base-sensor.md

```diff
@@ -1,12 +1,6 @@
-If you want to control your base by specifying a desired velocity in terms of distance/time, for example 5 m/s, and you have a sensor, for example a wheel encoder, that measures the actual velocity, you can make use of the sensor to adjust the velocity.
+If you want to control your base by specifying a desired velocity in terms of distance/time, for example 5 m/s, and you have a movement sensor that measures the actual velocity, you can make use of the sensor to adjust the velocity. Alternatively, if your base has position reporting motors, you can utilize the [wheeled odometry](../../../docs/components/movement-sensor/wheeled-odometry) movement sensor to get an estimate of the necessary velocities.
```

Comment:
thought I figured it out... but I did not. this link isn't passing the checks, let me know if you know of a better way to do this.